layouts/base.html

```diff
@@ -13,6 +14,8 @@
 </head>
 
 <body class="govuk-template__body  js-enabled">
+  <%= render '/partials/tag_manager_body.*' %>
```

Comment:
Also for when I'm working on the cookies stuff - why does this script need to be called from within the body rather than the head with the other google analytics script?

Reply:
The setup instructions specifically said to do this.

I was following the Google Tag Manager instructions [here](https://developers.google.com/tag-platform/security/guides/consent?consentmode=advanced#tag-manager_3).